### DISCUSSION TITLE
How to get git log with API?

### DISCUSSION THREAD

**Discussion by shinshinshin123** [community/community#34167] [Apps, API and Webhooks] [answered]
I don't know how to get the git audit log API.

It seems that it is possible to get the API if it is an enterprise plan.
I want to get the API outside of the enterprise plan.

Anyone know how?

**Comment by ghost** [comment, marked as answer]
https://github.com/jeviolle/github-audit-logs

**Reply by ettaboyle** [reply]
@shinshinshin123 did @BlankSpacePlus's response help solve your issue?

**Reply by shinshinshin123** [reply]
@ettaboyle @BlankSpacePlus
Sorry for the late reply.
It was very helpful.
Thank you very much.
